### PR TITLE
Update install.sh

### PR DESCRIPTION
**Pull request by ealmonte32**
initial draft changes for rebranding of the installer..
further changes such as `/home/${USER}/screenly` will require changes to all references of this directory (e.g. Ansible playbook, python code, shell scripts, etc.).

### CODE REVIEW

**Review by nicomiguelino — APPROVED** [review]
looks good to me

File: bin/install.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ if [ "$WEB_UPGRADE" = false ]; then
 
   # Make sure the command is launched interactive.
   if ! [ -t 0  ]; then
-    echo -e "Detected old installation command. Please use:\n$ bash <(curl -sL https://www.screenlyapp.com/install-ose.sh)"
+    echo -e "Detected old installation command. Please use:\n$ bash <(curl -sL https://install-ose.srly.io)"
```
Comment (vpetersson) [inline bin/install.sh:36]: I set up a new URL instead, which is `https://install-anthias.srly.io`. Could you update it?

**Comment by vpetersson** [comment]
LGTM too pending the URL change.

File: bin/install.sh (hunk 1 above)
Comment (ealmonte32) [reply bin/install.sh:36]: Sure.

**Comment by vpetersson** [comment]
Thanks @ealmonte32

**Pull request merged**